src/ble/att/handle.rs

```diff
@@ -10,7 +10,7 @@ use {
 ///
 /// The `0x0000` handle (`NULL`) is invalid and must not be used.
 #[derive(Copy, Clone)]
-pub struct AttHandle(u16);
+pub struct AttHandle(pub u16);
```

Comment:
I'd rather add a `fn from_raw(raw: u16) -> Self` constructor instead of making the field public

Reply:
Done